Makefile

```diff
@@ -139,30 +147,26 @@ clean:
 #   WHAT: Directory names to vet.  All *.go files under these
 #     directories will be vetted.  If not specified, "everything" will be
 #     vetted.
-#   TESTS: Same as WHAT.
-#   GOFLAGS: Extra flags to pass to 'go' when building.
-#   GOLDFLAGS: Extra linking flags to pass to 'go' when building.
 #
 # Example:
 #   make vet
 #   make vet WHAT=pkg/kubelet
 vet:
-	hack/verify-govet.sh $(WHAT) $(TESTS)
+	@hack/make-rules/vet.sh $(WHAT)
```

Comment:
Why not "verify-vet.sh"?
It seems to be a verification script, and we probably should be consistent with all other verify scripts...